integration-tests/src/cross_chain_transfer.rs

```diff
@@ -31,16 +31,17 @@ fn basilisk_should_receive_asset_when_transferred_from_relaychain() {
 		assert_ok!(basilisk_runtime::AssetRegistry::set_location(
 			basilisk_runtime::RuntimeOrigin::root(),
 			1,
-			basilisk_runtime::AssetLocation(MultiLocation::parent().into())
+			basilisk_runtime::AssetLocation(MultiLocation::parent())
 		));
 	});
 	Rococo::execute_with(|| {
-		assert_ok!(rococo_runtime::XcmPallet::reserve_transfer_assets(
+		assert_ok!(rococo_runtime::XcmPallet::limited_reserve_transfer_assets(
```

Comment:
reserve_transfer_asset was removed?

Reply:
deprecated, so I changed the test to get rid of the warning.